config/environments/development.rb

```diff
@@ -83,4 +84,11 @@
 
   # Devise config
   config.action_mailer.default_url_options = {host: "localhost", port: 3000}
+
+  alta_from_domain = "alta.localhost"
+  rubyforgood_from_domain = "rubyforgood.localhost"
+  config.email_from = {
+    default: {alta: "hello@#{alta_from_domain}", rubyforgood: "hello@#{rubyforgood_from_domain}"},
+    contact: {alta: "contact@#{alta_from_domain}", rubyforgood: "contact@#{rubyforgood_from_domain}"}
```

Comment:
We can probably store this on some sort of object or in the `Organization` table to avoid having to commit changes in order to onboard new organizations. I believe the more we have to commit code, the higher chances of breakage.

We aren't yet there yet so I think its fine to continue this way and can save that work for some other day if we see its needed. What ya think @nsiwnf ?

Reply:
Yeah thats a good idea. Maybe we can leave them in here until we talk a bit about how Organization is going to look?